monkestation/code/modules/viking/viking_axes.dm

```diff
@@ -0,0 +1,93 @@
+// fear the øx
+// øx means axe for the uncultured
+
+
+
+/obj/item/melee/viking
+	icon = 'monkestation/icons/viking/viking_items.dmi'
+	lefthand_file = 'monkestation/icons/viking/axes_lefthand.dmi'
+	righthand_file = 'monkestation/icons/viking/axes_righthand.dmi'
+	worn_icon = 'monkestation/icons/viking/viking_armor.dmi'
+
+/obj/item/melee/viking/tenja
+	name = "boarding axe"
+	icon_state = "hand_axe_item"
+	lefthand_file = "hand_axe_inhand_L"
+	righthand_file = "hand_axe_inhand_R"
+	worn_icon_state = "hand_axe_worn"
+	desc = "A one handed axe used by vikings."
+	force = 20
+	throwforce = 45
+	embedding = 50
+	wound_bonus = 25
+	slot_flags = ITEM_SLOT_BELT
+	w_class = WEIGHT_CLASS_NORMAL
+
+/obj/item/melee/viking/godly_tenja
+	name = "Leviathan Axe"
+	icon_state = "hand_axe_frost_item"
+	lefthand_file = "hand_axe_frost_inhand_L"
+	righthand_file = "hand_axe_inhand_R"
```

Comment:
Don't change the inhand_file. This is making it look for a file called "hand_axe_frost_inhand_L" not the actual sprites within. It won't work if you do it like this.